assets/css/learning-mode-compat.scss

```diff
@@ -0,0 +1,6 @@
+/*
+  This style files are only required when the current theme don't provide learning mode customizations.
+  Block based themes should declare add_theme_support('sensei-learning-mode'); to enable customization via theme.json
+*/
+@import 'sensei-course-theme/theme-fixes';
+@import 'sensei-course-theme/buttons';
```

Comment:
Should the styles setting fonts and some other stuff in [`base.scss`](https://github.com/Automattic/sensei/blob/de55b5263edc47e860c833af017849ecf7671514/assets/css/sensei-course-theme/base.scss#L15) be also moved here? That would probably make life simpler.

Reply:
Thanks for the suggestion @yscik <3 Moved the styles for the font family and link text decoration to the compat file [here](https://github.com/Automattic/sensei/pull/6066/commits/dfb237f9dd51e8475612ac1c4cb17eed9e97dff6), can you kindly take a look? Thanks

Reply:
Do you think that's enough, or would it make sense to move all the font formatting from there (size, weight, line height etc)? They are somewhat tied to the font. 

Similarly for the rest of the link styles: normal links are uncolored in the frame right now, but we could argue that's also a stylistic choice to be controlled by the theme. 

So pretty much everything aside from the variables could move from `base.scss`. But it's fine to only do that if it makes sense for the current use case.

Reply:
Thanks for the really good insights @yscik  💯 I was also in a dilemma about whether to move all the styles.  I didn't move some of the stylings from there as they're specified only by the `.sensei-course-theme__frame` class and general link tag state. So ideally all of them are overridable by theme.json, inline styles via template editor, or in some cases where necessary, a more specific styling So it felt that these styles kinda serve as a fallback unless they aren't easy to override or too opinionated. And also as you've indicated, hopefully, suffice for the current use case.